collaboration/learning_goals.md

```diff
@@ -1,5 +1,47 @@
 # Learning Goals
 
-## Collective
+## 🏆 Collective Objectives  
 
```

Comment:
Great work.. 
I would prefer if the individual goals listed alongside with name, to be easier to everyone to remember their goals.